web/src/pages/dashboard/index.page.jsx

```diff
@@ -1,57 +1,62 @@
-const Sidebar = () => {
-    
-   return (
-       <section className="bg-white dark:bg-gray-900">
+// Layout.js
 
+import { HiTemplate, HiOutlineShoppingBag, HiOutlineUsers } from "react-icons/hi";
+import Link from 'next/link';
 
-           <div className="flex flex-col items-center md:h-screen lg:py-0">
+const SidebarData = [
+  {
+    label: 'Dashboard',
+    icon: <HiTemplate />,
+    link: '/'
+  },
+  {
+    label: 'Orders',
+    icon: <HiOutlineShoppingBag />,
+    link: '/orders'
+  },
+  {
+    label: 'Staff',
+    icon: <HiOutlineUsers />,
+    link: '/staff'
+  }
+];
 
-               <h1 className='text-xl font-bold text-gray-800'>
-                   PC BEE ORDERING PARTS SYSTEM
-               </h1>
+const Layout = ({ children }) => {
+  return (
+    <div className="flex h-screen">
+      {/* Sidebar */}
+      <aside className="fixed top-0 left-0 z-40 w-64 h-screen bg-gray-50 dark:bg-gray-800">
```

Comment:
Since you put the SideBar and Navbar in Template.
You can call it like this:
```suggestion
    <Template>
         <main className="flex-1 overflow-x-hidden overflow-y-auto bg-gray-200 dark:bg-gray-800">
              Dashboard
          </main>
    
    </Template>
```